pkg/plugins/plugins.go

```diff
@@ -204,7 +204,11 @@ func (p *Plugin) implements(kind string) bool {
 func loadWithRetry(name string, retry bool) (*Plugin, error) {
 	registry := NewLocalRegistry()
 	start := time.Now()
-
+	var testTimeOut int
```

Comment:
I'm not a fan of this. Is there a specific reason to not override `defaultTimeOut` in the test file instead?

Reply:
Yeah, see the other comments  https://github.com/moby/moby/pull/46166/files#r1287038252. It's not "great" but at least this allows overriding for specific tests, and keeping the potential of using `t.Parallel()` without having to go through override variable -> defer reset variable loops.

Reply:
Would be great if there was some way to detect "are we being tested?" (`if runtime.TestActive ...`), although I guess that's a bit in conflict with "testing the actual" code as well.